frappe/migrate.py

```diff
@@ -119,9 +119,6 @@ def run_schema_updates(self):
 			skip_failing=self.skip_failing, patch_type=PatchType.pre_model_sync
 		)
 		frappe.model.sync.sync_all()
-		frappe.modules.patch_handler.run_all(
-			skip_failing=self.skip_failing, patch_type=PatchType.post_model_sync
-		)
 
```

Comment:
You can't just change semantics that have been in place for a long time like this. 

If a reload is required for a patch to work, then you must do `frappe.reload_doctype(...)` in the patch. There's no magic bullet for this problem. 


A reasonable compromise would be to introduce a new section of patches that run at the very end. But this change is absolutely not acceptable.